tests/io/test_eleanorlite.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from astropy.io import fits
+import numpy as np
+from numpy.testing import assert_array_equal
+
+from lightkurve import search_lightcurve
+from lightkurve.io.eleanorlite import read_eleanorlite_lightcurve
+from lightkurve.io.detect import detect_filetype
+
+@pytest.mark.remote_data
+def test_eleanor_lite():
+    """Can we read in GSFC-ELEANOR-LITE light curves?"""
+    url =
+    "https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HLSP/gsfc-eleanor-lite/s0001/0000/"
```

Comment:
Unfortuantely, the syntax for breaking down the url to 2 line isn't correct, causing the tests to fail:
https://github.com/lightkurve/lightkurve/runs/7080678164?check_suite_focus=true#step:5:35

For it to work, brackets are needed to surround the lines:
```python
url = (
    "line1"
    "line2"
)
```

Reply:
Thanks, syntax updated